src/sidebar/components/SidebarTabbedPanel.tsx

```diff
@@ -82,16 +91,15 @@ function Tab({
   );
 }
 
-export type SelectionTabProps = {
+export type SidebarTabbedPanelProps = {
   /** Are we waiting on any annotations from the server? */
   isLoading: boolean;
 
-  /** Counts of threads in each tab, to be displayed next to the tab title. */
-  tabCounts: {
-    annotation: number;
-    note: number;
-    orphan: number;
-  };
+  /**
+   * Can be used to hide all content except the thread list.
+   * Defaults to false.
+   */
+  threadListOnly?: boolean;
```

Comment:
This props is used to mimic the previous behavior in which the `ThreadList` could be rendered, but everything else in the tabs isn't.

I considered calling this `showTabs`, but it handles the rendering of the tabs and also some of the fallback content, so I thought it was not the right naming.

On the other hand `threadListOnly` leaks some internal implementation details, so it's also not perfect.